bin/main

```diff
@@ -1,72 +1,58 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-turn = 0
-player = true
-choices = [0]
-
-# Check for win and display winner
-def winner
-  puts 'Here we define if a winner is found if it is we display the winner'
-  false
+require_relative '../lib/board'
+require_relative '../lib/selection_display'
+
+$player_pick = ''
+$is_player_one_turn = true
+$turn = 0
+$board_controller = Board.new
+$local_selector = Selector.new
```

Comment:
Not global variables please.

Reply:
Can you direct me to where it is expressed on the project requirements we should not use global variables?

Reply:
Even though this isn't mention anywhere in the docs I can see why this is asked. 
Fixed